aspnet/client-side/bundling-and-minification.rst

```diff
@@ -1,10 +1,177 @@
+
+- Adding scripts by wildcard defaults to loading them in alphabetical order, which is typically not what you want. CSS and JavaScript files frequently need to be added in a specific (non-alphabetic) order. You can mitigate this risk by adding a custom  IBundleOrderer implementation, but explicitly adding each file is less error prone. For example, you might add new assets to a folder in the future which might require you to modify your  IBundleOrderer implementation.
+- View specific files added to a directory using wild card loading can be included in all views referencing that bundle. If the view specific script is added to a bundle, you may get a JavaScript error on other views that reference the bundle. 
+- CSS files that import other files result in the imported files loaded twice. 
+
+Bundle Considerations
+---------------------
+
+Bundling and minification primarily improve the first page request load time. Once a webpage has been requested, the browser caches the assets (JavaScript, CSS and images) so bundling and minification won’t provide any performance boost when requesting the same page, or pages on the same site requesting the same assets. If you don’t set the expires header correctly on your assets, and you don’t use bundling and minification, the browsers freshness heuristics will mark the assets stale after a few days and the browser will require a validation request for each asset. In this case, bundling and minification provide a performance increase after the first page request. For details, see the blog `Using CDNs and Expires to Improve Web Site Performance <http://blogs.msdn.com/b/rickandy/archive/2011/05/21/using-cdns-to-improve-web-site-performance.aspx>`_.
+
+The browser limitation of six simultaneous connections per each hostname can be mitigated by using a  CDN. Because the CDN will have a different hostname than your hosting site, asset requests from the CDN will not count against the six simultaneous connections limit to your hosting environment. A CDN can also provide common package caching and edge caching advantages.
+
+Gulp
+----
+Gulp is a JavaScript-based streaming build toolkit for client-side code. It is commonly used to stream client-side files through a series of processes when a specific event is triggered in a build environment. Gulp can specifically be used to automate the process of asset minification. Visual Studio's **ASP.NET 5 Web Application** project template includes a *gulpfile.js* file, which defines a set of Gulp tasks, including default tasks for minimizing JavaScript and CSS files during *design-time*. These tasks can be run from the **Task Runner Explorer** window in Visual Studio. For more information, see :doc:`using-gulp`.
+
+See Also
+--------
+- :doc:`using-gulp`
+- `Bundler & Minifier <https://visualstudiogallery.msdn.microsoft.com/9ec27da7-e24b-4d56-8064-fd7e88ac1c40>`_
```

Comment:
What do you want removed? The bundle considerations is great info and I got good feedback on that section.